### ISSUE TITLE
Multiple preferred syntax, or --accept-all?

### ISSUE THREAD

**Issue by yllan** [knopkem/dicom-dimse-native#20]
Hi,

Is there anyway to setup multiple accept transfer syntax or accept all? Thanks!

**Comment by knopkem** [comment]
Sorry for coming back so late. It's currently not possible but I will check if this could be added (at least accept-all should be simple)

**Comment by TickerCardiology** [comment]
Hi,
Do you know if the ability to accept-all was implemented? I'm familiar with the DCMTK tools and this would be an excellent addition.

**Comment by knopkem** [comment]
Hi, no it's not yet done. I would be happy to receive a PR for this.

**Comment by geezer78** [comment]
Hi All, i'd like to bump this question, would be very usefull to have accept-all as an option.

**Comment by knopkem** [comment]
@geezer78 ok, will have a look.

**Comment by geezer78** [comment]
Hi Michael, 
thanks for coming back so quickly!
I don't quite understand your answer however. 
Do you mean storescp and movescu should already be able to use all the mentioned (21)  transfer syntaxes? Thats not what I see happening, I see it's singing the defaults + the preferred.

**Comment by knopkem** [comment]
Yes, I thought that basically all 21 Ts should now be accepted, but apparently it doesn't work. Ok, will need to reproduce and try to find the issue with it. Thanks

**Comment by geezer78** [comment]
I'm leaving for a 1 week vacation now, will double check my results when I come back. Maybe I was wrong and it was another (retired) TS...

________________________________
